### DISCUSSION TITLE
[FR] Add lock\unlock button to editor toolbar

### DISCUSSION THREAD

**Discussion by cyb3rz3us** [standardnotes/forum#2620] [Suggestions]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**

**Describe the solution you'd like**
Add a button to the editor toolbar that allows locking\unlocking of a note.  A good example of this is the "Toggle Preview" button in the Markdown Pro editor.

Also, I would suggest a simple locked\unlocked padlock for the button's icon.

While the benefits are likely greater with the mobile apps, I personally see value in having this in both the mobile and desktop platforms.

**Comment by JaspalSuri** [comment]
I'm sorry if I've misunderstood this as a feature request rather than a suggestion on how to change the UI/UX of the existing feature. 

While the lock and unlock button appears in different places on the editor, it's available by going to <KBD>Options</KBD> > <KBD>Lock</KBD>.

**Comment by atmoio** [comment]
A locked note will also show a dedicated bar on the top of the screen with the option to quickly unlock (on web/desktop at least).

**Comment by cyb3rz3us** [comment]
@mobitar --- Thx for that pointer - definitely serves the same function for the unlock case in the desktop apps.

But in the case of the mobile apps --- which is where I personally would love to see this implemented --- it would be awesome to have a lock\unlock button right in the toolbar.  Just a simple toggle very much like the "Toggle Preview" button we have right now in the MD Pro editor.

I understand you can't be expected to implement every little thing and this definitely qualifies as a gravy kind of thing.  Still, I just wanted to toss it in the hat so to speak to see if maybe others see value in it.

Thanks.
